modules/ROOT/pages/developers/history.adoc

```diff
@@ -0,0 +1,57 @@
+= A bit of history
+
+Hawtio started as https://github.com/fusesource/fuse-console[Fuse Console] project to provide UI for Fuse (then moved to JBoss and Red Hat).
+
+Quickly it was rebranded as https://hawt.io/[Hawtio] and development continued with the same principles.
+
+The fundamental technologies and concepts used were:
+
+* Single Page Application model, where all routing happened at client (browser) side
+* https://jolokia.org[Jolokia] as https://docs.oracle.com/en/java/javase/17/jmx/jmx-technology-architecture.html[JMX Protocol Adaptor]
+  to expose MBean view of running Java/JVM application through web-friendly REST endpoints
+* https://www.typescriptlang.org/[TypeScript] as implementation language that adds type safety to JavaScript
+* https://angularjs.org/[Angular.JS] as implementation library/framework, based on data binding
+* https://getbootstrap.com/[Bootstrap] as CSS framework
+
+Initially all the development was performed in single https://github.com/hawtio/hawtio[Hawtio Github repository] and large part of the codebase was implemented in Java for the backend services. TypeScript code resided in single module and ultimately was packages as Java Web Archive (WAR).
```

Comment:
- Initially, _(comma)_
- in **a** single
- and **a** large part
- resided in **a** single
- packages as -> packaged as **a**